File: EP_02 Vetores/Exemplos/vetor_primo.py
def vetor_primo(vetor):
    for n, i in enumerate(vetor):
        if i == 2:
            vetor[n] = -1
        elif i > 1:
            for k in range(2, i + 1):
                if i % k == 0:
                    break
                else:
                    vetor[n] = -1
    print(vetor)


vetor_primo([2, 3, 4, 5, 6, 7])

File: EP_02/Exemplos/vetor_primo.c
#include <stdio.h>

void vetor_primo(int vetor[],int qtd){ 
        int i, div, x;
        for (i = 0; i < qtd; i++){ 
            for (div = 2; div < vetor[i]; div++){
                if (vetor[i] % div == 0){
                    break;
                }
                else{
                    vetor[i] = -1; 
                }
            }
        }
    for (x = 0; x < qtd; x++){ 
        printf("%d ",vetor[x]);
        }
    }

int main(){
    int vetor[6] = {2,3,4,5,6,7}, qtd = 6; 
    vetor_primo(vetor, qtd);
    return 0;
}
